<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tinklaRelayHudSettings</class>
 <widget class="QDialog" name="tinklaRelayHudSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="tinklaRelayHUD.qrc">
     <normaloff>:/img/cancel.png</normaloff>:/img/cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="tinklaRelayHUD.qrc">
     <normaloff>:/img/ok.png</normaloff>:/img/ok.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="speedLimitGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>461</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Speed Limit Type</string>
   </property>
   <widget class="QRadioButton" name="radioUS">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>US</string>
    </property>
    <property name="icon">
     <iconset resource="tinklaRelayHUD.qrc">
      <normaloff>:/img/speedLimitUS.png</normaloff>:/img/speedLimitUS.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioCA">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>CA</string>
    </property>
    <property name="icon">
     <iconset resource="tinklaRelayHUD.qrc">
      <normaloff>:/img/speedLimitCA.png</normaloff>:/img/speedLimitCA.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioROW">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>ROW</string>
    </property>
    <property name="icon">
     <iconset resource="tinklaRelayHUD.qrc">
      <normaloff>:/img/speedLimitEU.png</normaloff>:/img/speedLimitEU.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="orientationGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Orientation</string>
   </property>
   <widget class="QRadioButton" name="rotN">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="tinklaRelayHUD.qrc">
      <normaloff>:/img/rotationN.png</normaloff>:/img/rotationN.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotH">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="tinklaRelayHUD.qrc">
      <normaloff>:/img/rotationH.png</normaloff>:/img/rotationH.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotV">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="tinklaRelayHUD.qrc">
      <normaloff>:/img/rotationV.png</normaloff>:/img/rotationV.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotHV">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="tinklaRelayHUD.qrc">
      <normaloff>:/img/rotationHV.png</normaloff>:/img/rotationHV.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tinkla Relay HUD Settings</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="tinklaRelayHUD.qrc"/>
 </resources>
 <connections/>
</ui>
